README.md

```diff
@@ -115,6 +115,25 @@ final dashbook = Dashbook.multiTheme(
 ```
 
 ### Visibility control properties
+A `Dashbook` instance can also support localizations (i18n) and default language
+
+```dart
+final dashbook = Dashbook(
+    localizationsDelegates: [
+      AppLocalizations.delegate, // Custom Localization
+      GlobalMaterialLocalizations.delegate,
+      GlobalWidgetsLocalizations.delegate,
+    ],
+    supportedLocales: [
+      Locale('fr', ''),
+      Locale('en', ''),
+    ],
+  );
+
+```
+
+### Mobile example
+![Dashbook](https://user-images.githubusercontent.com/835641/88855642-9106e000-d1c9-11ea-8e5f-59e994a7fa03.gif)
```

Comment:
This section makes no sense, this gif is from a quite old version of the Dashbook

```suggestion
```